client/my-sites/checkout/checkout/privacy-protection.jsx

```diff
@@ -99,10 +103,132 @@ class PrivacyProtection extends Component {
 					isVisible={ this.props.isDialogVisible }
 					isFree={ hasOneFreePrivacy }
 					onSelect={ this.handleDialogSelect }
-					onClose={ this.handleDialogClose } />
+					onClose={ this.handleDialogClose }
+				/>
 			</div>
 		);
 	}
+
+	enablePrivacy = () => {
+		this.props.onRadioSelect( true );
+	};
+
+	disablePrivacy = () => {
+		this.props.onRadioSelect( false );
+	};
+
+	renderNoPopup() {
+		const domainRegistrations = cartItems.getDomainRegistrations( this.props.cart );
+		const { translate } = this.props;
+		const numberOfDomainRegistrations = domainRegistrations.length;
+
+		return (
+			<div>
+				<Card className="checkout__privacy-protection-radio">
+					<div>
+						{ translate(
+							'Domain owners have to share contact information in a public database of all domains. ' +
+							'With {{strong}}Privacy Protection{{/strong}}, we publish our own information instead of yours,' +
+							'and privately forward any communication to you.',
+							{
+								components: {
+									strong: <strong />
+								},
+							}
+						) }
+					</div>
+					<div className="checkout__privacy-protection-radio-buttons">
+						<FormFieldset>
+							<FormLabel className="checkout__privacy-protection-radio-button">
+								<FormRadio
+									value="private"
+									id="registrantType"
+									checked={ this.props.radioSelect === 'private' }
+									onChange={ this.enablePrivacy }
+								/>
+								<p className="checkout__privacy-protection-radio-text">
+									<span>
+										{
+											translate(
+												'{{strong}}Register privately with Privacy Protection{{/strong}} (recommended)' +
+												'',
+												{
+													components: {
+														strong: <strong />,
+													},
+												}
+											)
+										}
+									</span>
+									<span className={ 'checkout__privacy-protection-radio-price-text' }>
+										{
+											translate(
+												'%(cost)s/year',
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( '%(cost)s per year', '%(cost)s per domain per year' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).